qt_gui_app/package.xml

```diff
@@ -1,8 +1,9 @@
-<package>
+<package format="2">
   <name>qt_gui_app</name>
   <version>0.3.11</version>
   <description>
-    qt_gui_app provides the main to start an instance of the integrated graphical user interface provided by qt_gui.
+    qt_gui_app provides the main to start an instance of the integrated graphical user
+    interface provided by qt_gui.
```

Comment:
Please avoid unrelated changes like this to not diverge the branches more than necessary.